frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -286,6 +311,52 @@ const StrategyFormBody = ({
   );
 };
 
+const PreemptiveCachingSwitch = ({
+  enabled,
+  handleSwitchToggle,
+}: {
+  enabled?: boolean;
+  handleSwitchToggle: () => void;
+}) => {
+  const { values } = useFormikContext<CacheStrategy>();
+
+  const hasRefreshAutomatically = (
+    strategy: CacheStrategy,
+  ): strategy is DurationStrategy | ScheduleStrategy => {
+    return strategy.type === "duration" || strategy.type === "schedule";
+  };
+
+  const currentRefreshValue = hasRefreshAutomatically(values)
+    ? values.refresh_automatically
+    : false;
+
+  useEffect(() => {
+    if (hasRefreshAutomatically(values) && enabled !== currentRefreshValue) {
+      handleSwitchToggle();
+    }
+  }, [enabled, currentRefreshValue, handleSwitchToggle, values]);
+
+  return (
+    <Switch
+      checked={currentRefreshValue}
+      onChange={handleSwitchToggle}
+      role="switch"
+      size="sm"
+      label="Refresh cache automatically"
+      description="As soon as cached results expire, run and cache the query again to update
+      the results and refresh the cache."
+      styles={{
+        labelWrapper: { paddingLeft: "16px" },
+        label: { fontWeight: "bold" },
```

Comment:
@kdoh I wonder if this is a case where we should revisit the defaults for this switch. again this seems like a pretty "standard" switch

without styling | withstyling
---|---
![Screen Shot 2025-01-03 at 4 17 52 PM](https://github.com/user-attachments/assets/130acebf-13a7-4229-b69e-f4ba153e74ae) | ![Screen Shot 2025-01-03 at 4 17 41 PM](https://github.com/user-attachments/assets/b6ec307e-5052-4771-986f-a18a9a86f233)

@noahmoss , nothing for you to change here